### ISSUE TITLE
404 on Phonesky source

### ISSUE THREAD

**Issue by HPsaucii** [FriendlyNeighborhoodShane/MinMicroG#73]
Currently, your project uses pixel experience's phonesky source from gitlab, which has now been taken down. Therefore, any CI build releases that use phonesky have been broken for a while now, despite the latest manual release working, due to how old it is. I'd recommend switching to another source, to ensure future updates work. My suggestion *would* be nikgapps, but your scripts currently don't have support for the nikgapps zip distribution file structure. You could either add support, host the file somewhere (I did this method for local builds, by using http-server to host the extracted phonesky apk on my local network) or find a different source, idk.

Additionally, I'd recommend you add some more apps to your de-bloating lists, specifically:
```
  /system/system_ext/priv-app/GoogleFeedback/GoogleFeedback.apk
  /system/system_ext/priv-app/SetupWizard/SetupWizard.apk
  /system/system_ext/priv-app/GoogleServicesFramework/GoogleServicesFramework.apk

  /system/product/priv-app/GoogleOneTimeInitializer/GoogleOneTimeInitializer.apk
  /system/product/priv-app/GooglePartnerSetup/GooglePartnerSetup.apk
  /system/product/priv-app/Velvet/Velvet.apk

  /system/product/priv-app/Phonesky/Phonesky.apk
  /system/product/priv-app/GmsCore/GmsCore.apk
```
Removing these files means that the stock GMS doesn't remain on the system on modern stock samsungs, and the rest are for making sure that stock roms don't try to remove microG and install stock GMS again ( I don't know where these apps are stored on other phones, but the apps themselves should be present on all stock roms *somewhere* )

For reference, by making these relatively simple adjustments, adding signature spoofing support to my rom (i used whew-inc's Xposed module), and installing chiteroman's Play Integrity Fix, i can pass SafetyNet and basic PlayIntegrity. Although I'm unable to test it, you should be able to pass strong PlayIntegrity with osm0sis' Play Integrity Fix, provided you have a non-patched fingerprint.

**Comment by HPsaucii** [comment]
fixes Issue #64

**Comment by BlueFox964** [comment]
Hi, I saw your comment on my issue, and I was wondering if you would possibly make a fork of this project, with your fixes. I ended up just not using my new phone, and switching to my Oneplus because it had a vanilla ROM. I still have, and would really appreciate if you could!

**Comment by HPsaucii** [comment]
> Hi, I saw your comment on my issue, and I was wondering if you would possibly make a fork of this project, with your fixes. I ended up just not using my new phone, and switching to my Oneplus because it had a vanilla ROM. I still have, and would really appreciate if you could!

you can find the fork [here](https://github.com/HPsaucii/MinMicroG/tree/NikFixes)

**Comment by the-r3dacted** [comment]
With how long it's been, maybe should fork it, fix other issues and release builds.

**Comment by BlueFox964** [comment]
Hi saucii, I've built your fork, and tried using it, however, I still run into the same issue. My MicroG still dissapears. I'm on a chinese Coloros 14 devices, maybe thats the problem? I can't remove google apps with Sunils debloater either. I know this isn't your project, but I've looked literally everywhere, and I can't find a way to install MicroG on my device.

**Comment by HPsaucii** [comment]
> Hi saucii, I've built your fork, and tried using it, however, I still run into the same issue. My MicroG still dissapears. I'm on a chinese Coloros 14 devices, maybe thats the problem? I can't remove google apps with Sunils debloater either. I know this isn't your project, but I've looked literally everywhere, and I can't find a way to install MicroG on my device.

Did you use the modified steps for building as documented in my fork? It should build a properly working module, but im willing to do more work (such as making the nik fixes automatic)

**Comment by BlueFox964** [comment]
Yes, I did. I suspect it's because I'm on ColorOS, and when I try to install an older micog_installer_revived module, it comes up with an error saying gmscore is in the /my_bigball/ directory, instead of /data/

**Comment by HPsaucii** [comment]
@BlueFox964 Well, the best i could do is try to support your OS. If you could run a `find / -name "Phonesky.apk"` via a shell on your phone? If it doesnt return anything, try after running `su` aswell

**Comment by HPsaucii** [comment]
> With how long it's been, maybe should fork it, fix other issues and release builds.

Regarding this, you can build it yourself, its not hard. I might consider adding releases eventually. That repo i linked already contains all the fixes for issues im currently aware of

**Comment by BlueFox964** [comment]
How exactly would I do that? Is it through the use of Termux? Sorry if I'm being stupid.

But, afaik, it doesn't exist on my device, with it being Chinese and all that. I can offer the exact directories of the major play services though, being /my_bigball/priv-app/GmsCore, /my_bigball/priv-app/GooglePlayServicesUpdater (has the same package ID as the play store), and /my_bigball/non_overlay/priv-app/GoogleServicesFramework.

**Comment by HPsaucii** [comment]
> How exactly would I do that? Is it through the use of Termux? Sorry if I'm being stupid.
> 
> But, afaik, it doesn't exist on my device, with it being Chinese and all that. I can offer the exact directories of the major play services though, being /my_bigball/priv-app/GmsCore, /my_bigball/priv-app/GooglePlayServicesUpdater (has the same package ID as the play store), and /my_bigball/non_overlay/priv-app/GoogleServicesFramework.

Weird, seems like its just changing the /system directory to /my_bigball, ill see if i can modify the module to accomodate (no promises though)

**Comment by HPsaucii** [comment]
But yes, if you want to be doubly sure, you can use termux
